x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/user_flyout.tsx

```diff
@@ -92,22 +93,26 @@ export const UserFlyout: React.FC<Props> = ({
   );
 
   return (
-    <EuiFlyout ownFocus onClose={closeUserFlyout} size="s" aria-labelledby="userFlyoutTitle">
-      <EuiFlyoutHeader hasBorder>
-        <EuiTitle size="m">
-          <h2 id="userFlyoutTitle">{isComplete ? IS_COMPLETE_HEADING : IS_EDITING_HEADING}</h2>
-        </EuiTitle>
-        {!isComplete && (
-          <EuiText size="xs">
-            <p>{IS_EDITING_DESCRIPTION}</p>
-          </EuiText>
-        )}
-      </EuiFlyoutHeader>
-      <EuiFlyoutBody>
-        {children}
-        <EuiSpacer />
-      </EuiFlyoutBody>
-      <EuiFlyoutFooter>{isComplete ? completedFooterAction : editingFooterActions}</EuiFlyoutFooter>
-    </EuiFlyout>
+    <EuiPortal>
+      <EuiFlyout ownFocus onClose={closeUserFlyout} size="s" aria-labelledby="userFlyoutTitle">
```

Comment:
Nice catch on this EuiPortal!

Super optional, but as of EUI 34.x I believe EuiFlyout now defaults to `ownFocus={true}` so you could leave it out if you wanted. Don't worry if you'd rather not though, there's plenty of other flyouts in our app that still have ownFocus, so we could opt to clean it up across the board in a separate PR